src/pwr.rs

```diff
@@ -0,0 +1,169 @@
+//! Power configuration
+//!
+//! This module configures the PWR unit to provide the core voltage `VCORE`.
+//! The processor supports multiple voltage scaling modes, from VOS3 (lowest
+//! performance, lowest power consumption) to VOS0 (highest performance,
+//! highest power consumption).
+//!
+//! After reset the system is in VOS3. This power configuration module allows
+//! modes VOS0 to VOS3 to be selected (it defaults to VOS0 after calling
+//! `freeze`.
+//!
+//! ```rust
+//!     let dp = pac::Peripherals::take().unwrap();
+//!
+//!     let pwr = dp.PWR.constrain();
+//!     let pwrcfg = pwr.freeze();
```

Comment:
I would include a builder method in the example. Then in the documentation can state something like "if no mode is specified, then the default after calling `freeze` is VOS0"

```suggestion
//!     let pwrcfg = pwr.vos0().freeze();
```

Reply:
Good idea. I'll do so.